Procfile

```diff
@@ -1 +1 @@
-run-cron: ./erex_yomi.rb
+run-cron: bash -c "exec 1>$TOOL_DATA_DIR/out; exec 2>$TOOL_DATA_DIR/err; ./erex_yomi.rb"
```

Comment:
You can change the name of the log files to whatever you prefer (here they are `out` and `err`)